ros/ros

```diff
@@ -61,7 +61,7 @@ Architectures: amd64, arm64v8
 GitCommit: b525e9ef659ce448db6150fd5407ef62b2c5b265
 Directory: ros/jazzy/ubuntu/noble/ros-core
 
-Tags: jazzy-ros-base, jazzy-ros-base-noble, jazzy, latest
+Tags: jazzy-ros-base, jazzy-ros-base-noble, jazzy
```

Comment:
One image in the repo should still be tagged as `latest`, and we normally leave that to be the most recent LTS release (i.e. not rolling), and as of now it would still be Jazzy, no?

Reply:
Good catch - thanks!

Reply:
Done in d963117.